RDF-Pipeline/t/tests/0056_Set_Perl_Lib_Env_Var/expected-files/node/perlenv

```diff
@@ -5,27 +5,33 @@ use strict;
 # Verify that the Perl lib and other env vars are set properly 
 # to include RDF Pipeline libs.
 
+
 # The perl lib path:
+print "\nVerifying \@INC contains RDF-Pipeline path\n\n";
 my $inc = join(" ", @INC) || "";
-print "\@INC=$inc\n\n";
-$inc =~ m/Pipeline/ or die "Perl \@INC is not set properly: $inc\n";
+$inc =~ m/RDF-Pipeline\/lib/ or die "Perl INC is not set properly: $inc\n";
 
+print "Verifying PERL5LIB contains RDF-Pipeline/lib path\n\n";
 my $lib = $ENV{PERL5LIB} || "";
-print "PERL5LIB=$lib\n\n";
 $lib || die "\$PERL5LIB is not set!\n";
+$lib =~ m/RDF-Pipeline\/lib/ or die "Perl PERL5LIB is not set properly: $inc\n";
 
+print "Verifying RDF_PIPELINE_DEV_DIR contains an existing directory\n\n";
 my $dev = $ENV{RDF_PIPELINE_DEV_DIR} || "";
-print "RDF_PIPELINE_DEV_DIR=$dev\n\n";
 $dev || die "\$RDF_PIPELINE_DEV_DIR is not set!\n";
+-d $dev || die "RDF_PIPELINE_DEV_DIR does not exist";
 
-my $server = $ENV{SPARQL_SERVER} || "";
-print "SPARQL_SERVER=$server\n\n";
+# Disabled at David's recommendation since we may want to revisit this
+# SPARQL_SERVER is not set by default in set_env but it is used by some tools
+# my $server = $ENV{SPARQL_SERVER} || "";
+# print "SPARQL_SERVER=$server\n\n";
 # Okay if this isn't set:
 # $server || die "\$SPARQL_SERVER is not set!\n";
 
+print "Verifying that PATH contains RDF_PIPELINE_DEV_DIR\n\n";
 my $path = $ENV{PATH} || "";
-print "PATH=$path\n\n";
-$path =~ m/rdf-pipeline/ or die "\$PATH is not set!\n";
+(my $rdf_home = $dev) =~ tr#\\#\\//#;    # escape back slashes
```

Comment:
Suggestion instead of tr///: my $rdf_home = quotemeta($dev);
quotemeta ensures that all meta characters are escaped -- not just back slashes.  You might want ot make this same change elsewhere too.